goon-universe/app.js

```diff
@@ -252,7 +252,7 @@ router.route('/scenarios/initialAction')
             res.end()
         }
         else{
-        db.addInitActionChoice(studentID, scenarioID, data, function(result){
+        db.addInitActionResponse(studentID, scenarioID, data, function(result){
```

Comment:
`addInitActionResponse()` takes in more arguments than this.

Reply:
Fixed

Reply:
This has been implemented similar to addXReflectResponse functions with data being an object with questionID as key and choiceID as value